src/functions-and-arrays.js

```diff
@@ -52,14 +101,36 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  let uniqueArray = [];
+
+  wordsUnique.forEach(function(word){
+    if (uniqueArray.indexOf(word) === -1) {
+      uniqueArray.push(word);
+    } 
+  })
+
+  if (wordsUnique.length > 0){
+    return uniqueArray;
+  } else if (wordsUnique = ([])) {
+    return null;
+  }
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(wordsFind, wordToFind) {
+  for(let i = 0; 0 < wordsFind.length; i++){
+    if(wordsFind[i] === wordToFind) {
+      return "true";
+    } else {
+      return "false";
+    }
+  }
+}
 
```

Comment:
really good work, Zena! If you are interested, how to solve the bonus part, you can have a look at the solution.
For that function doesWordExist: instead of a string, you just send the boolean, like this:
```
if(wordsFind[i] === wordToFind) {
      return true;
```
otherwise it is only small things you have to change to have the test completely green. Such as:
```
 if (arrOfWords.length === 0) {
    return null;
  }
```
because the test says:
"should return null if receives an empty array when called"
But I think how you have to deal with the test you have understood very well.